### PR TITLE
add textAlignment option to the drawText method

### PR DESCRIPTION
**Pull request by TNGPS**
Simple refactor of the `PDFPage.drawText()` method allowing to choice from 3 defined textAlignments allready defined.

this solves #930

### CODE REVIEW

File: src/api/PDFPage.ts [hunk 1]
```diff
@@ -892,45 +889,52 @@ export default class PDFPage {
...
+    assertIsOneOfOrUndefined(options.textAlignment, 'options.allign', TextAlignment);
 
     const [originalFont] = this.getFont();
     if (options.font) this.setFont(options.font);
     const [font, fontKey] = this.getFont();
 
     const fontSize = options.size || this.fontSize;
 
-    const wordBreaks = options.wordBreaks || this.doc.defaultWordBreaks;
+    // const wordBreaks = options.wordBreaks || this.doc.defaultWordBreaks;
```
Comment (PhakornKiong) [inline src/api/PDFPage.ts:900]: Any reason why this is commented out and not removed?

File: src/api/PDFPage.ts [hunk 2]
```diff
@@ -892,45 +889,52 @@ export default class PDFPage {
     assertOrUndefined(options.maxWidth, 'options.maxWidth', ['number']);
     assertOrUndefined(options.wordBreaks, 'options.wordBreaks', [Array]);
     assertIsOneOfOrUndefined(options.blendMode, 'options.blendMode', BlendMode);
+    assertIsOneOfOrUndefined(options.textAlignment, 'options.allign', TextAlignment);
```
Comment (dsernst) [inline src/api/PDFPage.ts:892]: typo in `options.allign`, should be `.align`?

Comment (TNGPS) [reply src/api/PDFPage.ts]: thanks for the remark, this is now fixed in the latest version of the code

**Comment by Hopding** [comment]
Hello @TNGPS! Thanks so much for working on this! Apologies for my late response. I've been very busy these past few months taking care of some personal stuff and starting my new job at GitHub 🙂 

I'm hoping to get around to reviewing this PR in the next few weeks now that I've got some more free time.

Please note that you'll need to either update this PR (via a force push) or open a new PR with your changes. The tl;dr is that I recently had to do a fairly extensive git history rewrite that caused a bunch of conflicts. Please read https://github.com/Hopding/pdf-lib#git-history-rewrite for details. Apologies for the hassle!

**Comment by Hopding** [comment]
@TNGPS If you're still interested in getting this shipped, feel free to open a new PR (see my previous comment) and we can go from there 🙂

**Pull request closed without merging**